bin/ghe-restore

```diff
@@ -285,6 +285,16 @@ fi
 # Get GHES release version in major.minor format
 RELEASE_VERSION=$(ghe-ssh "$GHE_HOSTNAME" -- 'ghe-config --get core.package-version' | cut -d '.' -f 1,2)
 
+# If the backup being restored is from an appliance with Actions disabled, restoring it onto an appliance with Actions enabled will cause
+# mismatches in the secrets needed for Actions which ultimately results in Actions not working properly. Note: xargs is to remove whitespace
+ACTIONS_ENABLED_IN_BACKUP=$(git config -f $GHE_RESTORE_SNAPSHOT_PATH/settings.json app.actions.enabled | xargs)
+# If ACTIONS_ENABLED_IN_BACKUP is still empty, then set it to the default value of "false"
+ACTIONS_ENABLED_IN_BACKUP=${ACTIONS_ENABLED_IN_BACKUP:-"false"}
+if [[ $ACTIONS_ENABLED_IN_BACKUP != "true" ]] && ghe-ssh "$GHE_HOSTNAME" -- 'ghe-config --true app.actions.enabled'; then
```

Comment:
Nit: If you're going to check `!= "true"` rather than `== "false"`, then you probably don't need to assign "false" as default on line 292 anymore

Reply:
Good one, that was my intention but I did not commit it yet :)